<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dia_createUser</class>
 <widget class="QDialog" name="Dia_createUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sensores</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Seleccione el tiempo de 
medida para cada sensor 
de 5 a 20 segundos</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_temperatura">
          <property name="text">
           <string>Temperatura</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_temperatura">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_humedad">
          <property name="text">
           <string>Humedad</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_humedad">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_velocidadViento">
          <property name="text">
           <string>Velocidad del
Viento</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_velocidadViento">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_direccioViento">
          <property name="text">
           <string>Direcci칩n del
Viento</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBox_direccionViento">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_precipitacion">
          <property name="text">
           <string>Precipitaci칩n</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinBox_precipitacion">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_intensidadLuz">
          <property name="text">
           <string>Intensidad 
de luz</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="spinBox_intensidadLuz">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox_create">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>CREAR CUENTA</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_name">
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_name"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_lastName">
          <property name="text">
           <string>Apellido</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_lastName"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_document">
          <property name="text">
           <string>Documento</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_document"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_dateBirth">
          <property name="text">
           <string>Fecha 
Nacimiento</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDateEdit" name="dateEdit_dateBirth">
          <property name="locale">
           <locale language="Spanish" country="Colombia"/>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::DaySection</enum>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
          <property name="date">
           <date>
            <year>1972</year>
            <month>10</month>
            <day>20</day>
           </date>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_newUser">
          <property name="text">
           <string>Usuario</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_newUser"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_newPassword">
          <property name="text">
           <string>Contrase침a</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_newPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_confirmPassword">
          <property name="text">
           <string>Confirmar 
Contrase침a</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_confirmPassword"/>
        </item>
        <item row="8" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="1">
         <widget class="QComboBox" name="comboBox_cultivo">
          <item>
           <property name="text">
            <string>Papa</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rosas</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_cultivo">
          <property name="text">
           <string>Cultivo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_validar">
     <property name="text">
      <string>Validar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_create</sender>
   <signal>accepted()</signal>
   <receiver>Dia_createUser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_create</sender>
   <signal>rejected()</signal>
   <receiver>Dia_createUser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
